section2/exercises/if-statements.js

```diff
@@ -65,8 +72,18 @@ When numQuarters = 3, program should log "I have enough money for a gumball"
 var numQuarters = 0;
 
 // Write your conditional here
-console.log("I have enough money for a gumball");
-console.log("I don't have enough money for a gumball");
+
+if (numQuarters === 0) {
```

Comment:
Similar to the push on the pizza problem, could you use some of the tools you've learned to reduce the amount of redundant code?